lnwallet/rpcwallet/rpcwallet.go

```diff
@@ -159,6 +159,10 @@ func (r *RPCKeyRing) SendOutputs(outputs []*wire.TxOut,
 			return nil, fmt.Errorf("error looking up utxo: %v", err)
 		}
 
+		if txscript.IsPayToTaproot(info.PkScript) {
+			signDesc.HashType = txscript.SigHashDefault
```

Comment:
Why do we need to override this? As we assume that only default should ever be used here? Or does something else get tripped up by the 65 byte sig? If we rebase this on master (which now includes https://github.com/lightningnetwork/lnd/pull/7332), then can we remove this?

Reply:
This is changing the sighash type from `txscript.SigHashAll` to `txscript.SigHashDefault` in case it's a Taproot output. So in case one of the inputs was a Taproot input, this wouldn't have worked before. I'm not sure why this wasn't discovered in a test previously, maybe it's just a case we haven't covered in the remote signer tests yet.